src/modules/users/components/Inbox/InboxContainer/InboxContainer.tsx

```diff
@@ -51,7 +51,7 @@ const allReadActions = {
 };
 
 const InboxContainer = ({ full, close }: Props) => {
-  const inboxItems = useSelector(inboxItemsSelector);
+  const { record: inboxItems } = useSelector(inboxItemsSelector);
```

Comment:
Yeah actually here's a case where we don't convert to a JS object I think? Appears to work fine in this component though.

Reply:
I always thought `useSelector` would do the conversion and pick the `record`

Reply:
Oh wow it does! Okay that's really confused me 😵 the react-redux hook is also called `useSelector` (and that doesn't do any extra processing).

Reply:
So why doesn't it do it in this case?

Reply:
No it does, I was thinking `useSelector` was the `react-redux` one.

Reply:
But then why do we need to do it here? Is it a nested `record` for some reason?

Reply:
Oh right, no it doesn't pick `record` just does the `toJS` conversation. Sorry, I was reading/replying too fast!